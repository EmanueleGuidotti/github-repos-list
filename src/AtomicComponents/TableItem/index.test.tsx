import React from "react";
import renderer from "react-test-renderer";
import TableItemComponent from "./index";

describe("TableItemComponent", () => {
  it("Should render item component", () => {
    const component = renderer.create(<TableItemComponent />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("Should render item component with children", () => {
    const component = renderer.create(
      <TableItemComponent children={<p>I'm a children</p>} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("Should render item component with an external style and children", () => {
    const component = renderer.create(
      <TableItemComponent
        otherStyle={{ justifyContent: "start" }}
        children={<p>I'm a children</p>}
      />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
