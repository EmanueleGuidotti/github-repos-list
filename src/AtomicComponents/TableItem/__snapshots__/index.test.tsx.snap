// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableItemComponent Should render item component 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flex": "1",
      "paddingBottom": "20px",
      "paddingLeft": "30px",
      "paddingTop": "20px",
    }
  }
/>
`;

exports[`TableItemComponent Should render item component with an external style and children 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flex": "1",
      "justifyContent": "start",
      "paddingBottom": "20px",
      "paddingLeft": "30px",
      "paddingTop": "20px",
    }
  }
>
  <p>
    I'm a children
  </p>
</div>
`;

exports[`TableItemComponent Should render item component with children 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flex": "1",
      "paddingBottom": "20px",
      "paddingLeft": "30px",
      "paddingTop": "20px",
    }
  }
>
  <p>
    I'm a children
  </p>
</div>
`;
